app/src/main/java/fr/free/nrw/commons/media/MediaDetailPagerFragment.java

```diff
@@ -88,17 +95,6 @@ public static MediaDetailPagerFragment newInstance(boolean editable, boolean isF
         return mediaDetailPagerFragment;
     }
 
-    public static MediaDetailPagerFragment newInstance(boolean editable, boolean isFeaturedImage,
```

Comment:
This was being called only from MediaDetailPagerFragmentUnitTests. So removed it as this was not being called by anywhere in the app.